src/patterns/components/promo/base.hbs

```diff
@@ -14,7 +14,7 @@ description:
   {{#if promoIcon}}
   <a href="#" class="sprk-o-Stack__item">
     <svg height="150" width="167" viewBox="0 0 220.63 197.62">
-      <use xlink:href="#sparkler" />
+      <use xlink:href="#bell" />
```

Comment:
i removed the old spark icons that i had created so this is updating that old path to not be a broken icon